src/src/Commands/Tasks.cpp

```diff
@@ -51,7 +51,7 @@ String Command_Task_Disable(struct EventStruct *event, const char *Line)
   unsigned int varNr;
   String dummy;
 
-  if (validTaskVars(event, taskIndex, varNr) && PluginCall(PLUGIN_EXIT, event, dummy) && setTaskEnableStatus(taskIndex, false)) {
+  if (validTaskVars(event, taskIndex, varNr) && (PluginCall(PLUGIN_EXIT, event, dummy) || true) && setTaskEnableStatus(taskIndex, false)) {
```

Comment:
`validTaskVars(event, taskIndex, varNr) && (PluginCall(PLUGIN_EXIT, event, dummy) || true)`
This is technically correct, but it is also typically a piece of code that may later end up in a bug as it is not immediately clear what it does.

In this particular case I would use nested if statements instead of the && syntax.

Reply:
Reworked the code for readability